app/assets/javascripts/incidents.subquestions.js

```diff
@@ -75,4 +78,19 @@ $(function(){
     $('#multiple-agencies-info-text').toggle($('#screener_multiple_agencies_true').is(':checked'));
     $('#shots-fired-info-text').toggle($('#screener_shots_fired_true').is(':checked'));
   }).change();
+
+
+  function medicalAid() {
+    $('.medical-aid-received input[type="radio"]').on('click', function () {
+      togglePreExistingCondition()
+    })
+  }
+
+  function togglePreExistingCondition() {
```

Comment:
You can just put all this text within the on.click function body. Like before, I don't see a reason to wrap this code in an extra function that is just invoked, creates unnecessary indirection.